### PR TITLE
fix(artist): Ensure subroutes 404 by adding @principleField directive

### PR DESCRIPTION
**Pull request by damassi**
The type of this PR is: **Fix**

Addresses [DIA-745]

### Description

This fixes a (somewhat mysterous) issue where subroutes that don't match don't redirect to 404.

Looking through other routes, we add principleField on subroutes. So it looks like this was a long-term issue that we've just discovered, that was perhaps being masked somehow pre router upgrade.

[DIA-745]: https://artsyproduct.atlassian.net/browse/DIA-745?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/ArtworkFilter/Utils/getInitialFilterState.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Location } from "found"
-import { ArtworkFilters } from "../ArtworkFilterContext"
+import { ArtworkFilters } from "Components/ArtworkFilter/ArtworkFilterContext"
```
Comment (damassi) [inline src/Components/ArtworkFilter/Utils/getInitialFilterState.ts:2]: Also, per previous convo, incrementalism at work (eslint discussion) 🤷 Works good.

File: src/Apps/Artist/Routes/CV/ArtistCVRoute.tsx [hunk 2]
```diff
@@ -42,14 +46,20 @@ export const ArtistCVRouteFragmentContainer = createFragmentContainer(
   {
     viewer: graphql`
       fragment ArtistCVRoute_viewer on Viewer {
-        soloShows: artist(id: $artistID) {
+        soloShows: artist(id: $artistID)
+          @principalField
```
Comment (mzikherman) [inline src/Apps/Artist/Routes/CV/ArtistCVRoute.tsx:50]: Does this work? I thought MP will loudly complain if you pass in a query with more than one `@principalField` directive declared (that's sort-of implied with the 'principal' naming -> can be only one).

**Review by mzikherman — APPROVED** [review]
I'd merge this w/o the shows + CV query updates as they _seem_ like it may not work as expected (due to using `@principalField` multiple times) - not sure if you hit those routes locally.

**Comment by damassi** [comment]
I did not -- but i can update 👍

**Comment by mzikherman** [comment]
oo - or maybe just pick one of the groups in the shows/CV tab and add the directive to that one, versus all of them.

File: src/Apps/Artist/Routes/CV/ArtistCVRoute.tsx (hunk 2 above)
Comment (damassi) [reply src/Apps/Artist/Routes/CV/ArtistCVRoute.tsx]: fixed!

Comment (mzikherman) [reply src/Apps/Artist/Routes/CV/ArtistCVRoute.tsx:50]: Sweet - thanks!

**Pull request merged**